spec/unit/artifactserver_spec.rb

```diff
@@ -56,6 +56,59 @@ module CookbookOmnifetch
       }
       expect(public_repo_location.lock_data).to eq(expected_data)
     end
+
+    context "when asked to install a new cookbook" do
+
+      let(:http_client) { double("Http Client") }
```

Comment:
Yeah, I figured that something funky was going on.  I tried using instance double but it kept failing to find CookbookCmnifetch::Chef.  I figured that in this case, those namespaces/types were being pulled in from somewhere else - so I created a bare double instead.